packages/spectral/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@prismatic-io/spectral",
-  "version": "10.6.1",
+  "version": "10.7.0",
```

Comment:
Let me know if we think this should be major.

Reply:
I think this should probably be a major given it will break certain consumers.